tests/components/matter/test_light.py

```diff
@@ -20,7 +20,7 @@ async def light_node_fixture(
 ) -> MatterNode:
     """Fixture for a light node."""
     return await setup_integration_with_node_fixture(
-        hass, "dimmable-light", matter_client
+        hass, "extended-color-light", matter_client
```

Comment:
In a follow up PR, I think we should split the light tests so that we can test all lights that support on, off and brightness in one test via `pytest.mark.parametrize` and different fixtures, and another test for the color lights, etc.

Reply:
Perfect I'll make tests for each light type :)